Meadowlark-js/packages/meadowlark-core/src/message/DeleteRequest.ts

```diff
@@ -7,7 +7,8 @@ import { ResourceInfo } from '../model/ResourceInfo';
 import { Security } from '../security/Security';
 
 export type DeleteRequest = {
-  id: string;
+  documentUuid: string;
+  meadowlarkId: string;
```

Comment:
There is no body for a delete request, so there is no document to extract a DocumentIdentity from, so the meadowlarkId that can be computed is useless (only has the resource information). It should be removed from DeleteRequest.